SolrNet/Impl/SolrBasicServer.cs

```diff
@@ -142,26 +142,32 @@ public Task<ResponseHeader> AtomicUpdateAsync(string uniqueKey, string id, IEnum
             return SendAndParseHeaderAsync(atomicUpdate);
         }
 
-        public ResponseHeader Ping() {
-            return SendAndParseHeader(new PingCommand());
+        public ResponseHeader Ping() {
+            return SendAndParseHeader(new PingCommand());
         }
 
         public SolrSchema GetSchema(string schemaFileName) {
-            string schemaXml = connection.Get("/admin/file", new[] { new KeyValuePair<string, string>("file", schemaFileName) });
-            var schema = XDocument.Parse(schemaXml);
-            return schemaParser.Parse(schema);
+            string schemaXml = connection.Get("/admin/file", new[] { new KeyValuePair<string, string>("file", schemaFileName) });
+            var schema = XDocument.Parse(schemaXml);
+            return schemaParser.Parse(schema);
+        }
+
+        public LukeResponse GetLuke() {

```

Comment:
1. This should be part of an interface, for example `ISolrBasicReadOnlyOperations` like `GetSchema`
2. Also add the `Async` version.